[2024-08-09T09:28:44.747+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-09T09:28:44.764+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_2024-08-09.log_the_data manual__2024-08-09T09:28:35.422372+00:00 [queued]>
[2024-08-09T09:28:44.768+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_2024-08-09.log_the_data manual__2024-08-09T09:28:35.422372+00:00 [queued]>
[2024-08-09T09:28:44.769+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-09T09:28:44.858+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): log_the_data> on 2024-08-09 09:28:35.422372+00:00
[2024-08-09T09:28:44.866+0000] {standard_task_runner.py:63} INFO - Started process 12938 to run task
[2024-08-09T09:28:44.873+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_2024-08-09', 'log_the_data', 'manual__2024-08-09T09:28:35.422372+00:00', '--job-id', '2333', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpavub5mof']
[2024-08-09T09:28:44.877+0000] {standard_task_runner.py:91} INFO - Job 2333: Subtask log_the_data
[2024-08-09T09:28:45.269+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_2024-08-09.log_the_data manual__2024-08-09T09:28:35.422372+00:00 [running]> on host 05bc191a9412
[2024-08-09T09:28:46.220+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_2024-08-09' AIRFLOW_CTX_TASK_ID='log_the_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-09T09:28:35.422372+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-09T09:28:35.422372+00:00'
[2024-08-09T09:28:46.222+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-09T09:28:46.239+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-09T09:28:46.240+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/logging_config.py']
[2024-08-09T09:28:46.245+0000] {subprocess.py:86} INFO - Output:
[2024-08-09T09:28:46.324+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-09T09:28:46.325+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-09T09:28:46.361+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_2024-08-09, task_id=log_the_data, run_id=manual__2024-08-09T09:28:35.422372+00:00, execution_date=20240809T092835, start_date=20240809T092844, end_date=20240809T092846
[2024-08-09T09:28:46.413+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-09T09:28:46.459+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-09T09:28:46.512+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
